theme/style.css

```
@@ -0,0 +1,513 @@
+  --kb-secondary:     #B50232;
+  --kb-accent:   #DC0037;
+  --kb-highlight:    #FF780F;
+  --kb-dark:    #131010;
+  --kb-light:    #FFFFFF;
+  --kb-soft:     #fce8ee;
+  --kb-muted:     #f5f5f5;
+  --kb-border:    #e5e5e5;
+  --radius:         6px;
+  --shadow-sm:      0 1px 4px rgba(0,0,0,0.10);
+  --shadow-md:      0 4px 16px rgba(0,0,0,0.12);
+  --font:           'Brave Sans', -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)